apps/mark/frontend/src/pages/idle_page.tsx

```diff
@@ -27,6 +28,7 @@ export function IdlePage(): JSX.Element {
   useEffect(() => {
     async function reset() {
       setIsLoading(true);
+      await sleep(1000); // Allow "Clearing ballot" text to display
```

Comment:
Needed to add this since `markVoterCardVoided` below is now instantaneous (it previously involved an async card write)